js/graviex.js

```diff
@@ -0,0 +1,987 @@
+                    ],
+                    'post': [
+                        'orders', // Requires access_key, tonce, signature and api secret AND market, side(sell/buy), volume. Optional price ord_type
+                        'orders/multi', // Requires access_key, tonce, signature and api secret AND market, orders, orders[side(sell/buy)], orders[volume]. Optional orders[price], orders[ord_type]. To create multiple orders at once
+                        'orders/clear', // Requires access_key, tonce, signature and api secret. Optional side(sell/buy) to only clear orders on one side.
+                        'order/delete', // Requires access_key, tonce, signature and api secret AND id (unique orderid).
+                        'account/store', // Not Using Currently In Unified Methods.
+                        'create_fund_source', // Not Using Currently In Unified Methods.
+                        'remove_funs_source', // Not Using Currently In Unified Methods.
+                        'strategy/cancel', // Not Using Currently In Unified Methods.
+                        'strategy/create', // Not Using Currently In Unified Methods.
+                        'strategy/update', // Not Using Currently In Unified Methods.
+                        'strategy/activate', // Not Using Currently In Unified Methods.
+                        'strategy/deactivate', // Not Using Currently In Unified Methods.
+                    ],
+                },
+            },
+            'fees': {
+            },
+            'limits': {
+                'amount': {
+                    'min': 0.001,
+                    'max': undefined,
+                },
+            },
+            'precision': {
+                'amount': 8,
+                'price': 8,
+            },
+            'exceptions': {
+                'exact': {
+                    '2002': InvalidOrder, // {"error":{"code":2002,"message":"Failed to create order. Reason: Volume is too small"}}
+                    '1001': ExchangeError, // {"error":{"code":1001,"message":"market does not have a valid value"}}
+                    '2008': AuthenticationError, // {"error":{"code":2008,"message":"The access key XXXXXXXXXXXXXXXXXXXX does not exist."}}
+                },
+            },
+        });
+    }
+
+    async fetchTime (params = {}) {
+        const response = await this.publicGetTimestamp (params);
+        // 1644821226
+        return response;
+    }
+
+    async fetchMarkets (params = {}) {
+        const response = await this.publicGetMarkets ();
+        // {
+        //     "id": "giobtc",
+        //     "name": "GIO/BTC",
+        // };
+        const markets = response;
+        const result = [];
+        for (let i = 0; i < markets.length; i++) {
+            const market = markets[i];
+            const id = this.safeString (market, 'id');
+            const symbol = this.safeString (market, 'name');
+            const [ baseId, quoteId ] = symbol.split ('/');
```

Comment:
let's make `baseId` and `quoteId` lower case here and then we can remove the `toLowerCase` calls in the requests

Reply:
Okay. Fixed in the latest commit https://github.com/ccxt/ccxt/commit/81c8152ac30b15da7aad6a3f117da386d51efd62